aten/src/ATen/native/native_functions.yaml

```diff
@@ -1773,17 +1773,15 @@
 - func: type_as(Tensor self, Tensor other) -> Tensor
   variants: method
 
-- func: _unique(Tensor self, bool sorted=false, bool return_inverse=false) -> (Tensor, Tensor)
+- func: unique(Tensor self, bool sorted=false, bool return_inverse=false, int64_t? dim=None) -> (Tensor, Tensor)
   variants: function
   dispatch:
     CPU: _unique_cpu
     CUDA: _unique_cuda
 
-- func: _unique_dim(Tensor self, int64_t dim, bool sorted=false, bool return_inverse=false) -> (Tensor, Tensor)
+# unique is not exposed to python, special cased in gen_python_functions.py
```

Comment:
you mean unique_dim is not exposed to python?

Reply:
Yes, thanks for figuring out